### PR TITLE
Update PRR process status

### PR DESCRIPTION
**Pull request by johnbelamaric**
Follow up to https://github.com/kubernetes/enhancements/pull/1620, this removes the questionnaire from here and notes that PRR is required in 1.19.

### CODE REVIEW

**Comment by johnbelamaric** [comment]
/hold
For merge of https://github.com/kubernetes/enhancements/pull/1620

**Comment by mrbobbytables** [comment]
@johnbelamaric https://github.com/kubernetes/enhancements/pull/1620 has merged. Should the hold be lifted and this merged?

**Comment by johnbelamaric** [comment]
yes, thanks for the reminder

/hold cancel

File: sig-architecture/production-readiness.md [hunk 1]
```diff
@@ -5,170 +5,21 @@ Kubernetes are observable, scalable and supportable, can be safely operated in
...
-  are missing a bunch of machinery and tooling and do that now.
-
-
-#### Monitoring requirements
-
-* **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metrics. Operations against Kubernetes API (e.g.
-  checking if there are objects with field X set) may be last resort. Avoid
-  logs or events for this purpose.
+More details may be found in the [PRR KEP][].
```
Comment (justaugustus) [inline sig-architecture/production-readiness.md:8]:
```suggestion
More details may be found in the [PRR KEP].
```

File: sig-architecture/production-readiness.md [hunk 2]
```diff
@@ -5,170 +5,21 @@ Kubernetes are observable, scalable and supportable, can be safely operated in
...
-      Not required until feature graduated to Beta.
-    - Testing: Are there any tests for failure mode? If not describe why.
+## Status
 
-* **What steps should be taken if SLOs are not being met to determine the problem?**
+As of 1.19, production readiness reviews are required, and are part of the KEP
+process. The PRR questionnaire previously found here has been incorporated into
+the [KEP template]. The template details the specific questions that must be
+answered, depending on the stage of the feature. As of 1.19, PRRs are
+non-blocking; that is, _approval_ is not required for the feature to be part of
```
Comment (justaugustus) [inline sig-architecture/production-readiness.md:16]:
```suggestion
non-blocking; that is, _approval_ is not required for the enhancement to be part of
```

**Pull request merged**